android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -190,21 +190,27 @@ public String getCarrier() {
   }
 
   @ReactMethod
-  public Integer getTotalDiskCapacity() {
+  public long getTotalDiskCapacity() {
```

Comment:
I think you should use "Long" instead of "long". I am having the same issue and tried using your fix.
Android is complaining with _"incompatible types: <null> cannot be converted to long"_